instagram/src/App.css

```diff
@@ -0,0 +1,26 @@
+
+.App {
+  text-align: center;
+  padding: 0 100px;
+}
+
+ .App-logo {
+  animation: App-logo-spin infinite 20s linear;
+  height: 40vmin;
+  pointer-events: none;
+}
+
+ .App-header {
+  background-color: #282c34;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
+
+ .App-link {
+  color: #61dafb;
+}
```

Comment:
you can remove all the starter styles they added in the boilerplate and set up your own. remember to use resets and utilize the responsive methods we learned like setting our box-sizing: border-box and setting img {width: 100%; height: auto} etc